Dockerfile.ocp

```diff
@@ -20,4 +20,14 @@ RUN dnf update -y && \
     dnf clean all && \
     rm -rf /var/cache/{yum,dnf}/*
 
+FROM ubi8
+
+RUN dnf update -y && \
```

Comment:
Why do we even need this any more?

Reply:
we actually don't and it's an anti-pattern anyway, I will remove it